third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -76,29 +87,67 @@ static Value cvtFp16ToFp32(Location loc, ConversionPatternRewriter &rewriter,
 }
 
 static Value cvtFp32ToFp16(Location loc, ConversionPatternRewriter &rewriter,
-                           const Value &v) {
+                           const Value &v, const RoundingMode rounding) {
   GCNBuilder builder;
+
   auto &cvt = *builder.create("v_cvt_f16_f32");
   auto res = builder.newOperand("=v");
   auto operand = builder.newOperand(v, "v");
+  if (rounding == RoundingMode::RTZ) {
+    auto &setRTZ = *builder.create("s_setreg_imm32_b32 0x1801, 0xc");
+    setRTZ();
+  }
   cvt(res, operand);
+  if (rounding == RoundingMode::RTZ) {
+    auto &resetRTZ = *builder.create("s_setreg_imm32_b32 0x1801, 0x0");
+    resetRTZ();
+  }
   return builder.launch(rewriter, loc, f16_ty, false);
 }
 
-static SmallVector<Value>
-convert_val_Fp16_to_Fp8(Location loc, ConversionPatternRewriter &rewriter,
-                        Value v0, Value v1, const std::string &fp8_format) {
+// convert fp8 to fp32
+static SmallVector<Value> cvtFp8ToFp32(Location loc,
+                                       ConversionPatternRewriter &rewriter,
+                                       Value v0, Value v1,
+                                       const std::string &fp8_format) {
   assert(fp8_format == "fp8" or fp8_format == "bf8");
```

Comment:
Ha, this `or` is python style. :) In C++ we use `||` commonly right?

Reply:
Done.